parser-typechecker/src/Unison/Builtin.hs

```diff
@@ -2,75 +2,89 @@
 {-# LANGUAGE Rank2Types #-}
 
 module Unison.Builtin
-  (codeLookup
-  ,constructorType
-  ,names
-  ,names0
-  ,builtinDataDecls
-  ,builtinEffectDecls
-  ,builtinConstructorType
-  ,builtinTypeDependents
-  ,builtinTypeDependentsOfComponent
-  ,builtinTypes
-  ,builtinTermsByType
-  ,builtinTermsByTypeMention
-  ,intrinsicTermReferences
-  ,intrinsicTypeReferences
-  ,isBuiltinType
-  ,typeOf
-  ,typeLookup
-  ,termRefTypes
-  ) where
-
-import Unison.Prelude
-
-import           Data.Bifunctor                 ( second, first )
-import qualified Data.Map                      as Map
-import qualified Data.Set                      as Set
-import qualified Data.Text                     as Text
-import qualified Text.Regex.TDFA               as RE
-import Unison.ConstructorReference (GConstructorReference(..))
-import qualified Unison.ConstructorType        as CT
-import           Unison.Codebase.CodeLookup     ( CodeLookup(..) )
-import qualified Unison.Builtin.Decls          as DD
-import qualified Unison.Builtin.Terms          as TD
-import qualified Unison.DataDeclaration        as DD
+  ( codeLookup,
```

Comment:
Huh, weird this didn't get ormolu'd already. 🤷🏼

Reply:
I guess it needed a little nudge.